tests/sys.rs

```diff
@@ -134,3 +173,36 @@ mod policy {
         assert!(resp.is_ok());
     }
 }
+
+// Sets up a new test using the vault production server.
+pub fn new_prod_test() -> Test {
+    let mut test = Test::default();
+    let vault_config = serde_json::json!({
+        "listener": [
+            {
+                "tcp": {
+                    "address": "0.0.0.0:8300",
+                    "tls_disable": "true"
+                }
+            }
+        ],
+        "storage": [
+            {
+                "file": {
+                    "path": "/vault/file"
+                }
```

Comment:
We should use the inmem storage backend here
```suggestion
                "inmem": {}
```